### ISSUE TITLE
Importing Legra in script tag requires to use lowercase constructor

### ISSUE THREAD

**Issue by r34117y** [pshihn/legra#5]
```const legra = new Legra(ctxFace, 20);```
```Uncaught ReferenceError: Legra is not defined```

```const legra = new legra(ctxFace, 20);```
```Uncaught ReferenceError: Cannot access 'legra' before initialization```

```const legra = new legra(ctxFace, 20);```
Above works ok

It is not mentioned in docs :)

**Comment by pshihn** [comment]
Yep it should be Capitalized in this case, because it's a class name. 
Thanks for reporting
